File: app/src/at-myhuynh/java/asiantech/internship/summer/java_core/ex06/ListPrimeFirst.java
package asiantech.internship.summer.java_core.ex06;


import java.util.logging.Logger;

import asiantech.internship.summer.java_core.common.Common;

public class ListPrimeFirst {

    public static void main(String[] args) {
        int n = Common.input("Nhập vào n: ");
        print(n);
    }

    public static void print(int n) {
        Logger log = Logger.getLogger(ListPrimeFirst.class.getName());
        int count = 0;
        int number = 1;

        do {
            if (Common.isPrime(number)) {
                count++;
                log.info(String.valueOf(number));
            }
            number++;
        } while (count < n);
    }

}

File: app/src/at-myhuynh/java/asiantech/internship/summer/kotlin/ex06/ListPrimeFirst.kt
package asiantech.internship.summer.kotlin.ex06

import asiantech.internship.summer.kotlin.common.Common
import asiantech.internship.summer.kotlin.common.isPrime

object ListPrimeFirst {

    @JvmStatic
    fun main(args: Array<String>) {
        val n = Common.input("Nhập vào n: ")
        print(n)
    }

    private fun print(n: Int) {
        var i = n
        var number = 1

        do {
            if (number.isPrime()) {
                i--
                println(number)
            }
            number++
        } while (i > 0)
    }
}
